### DISCUSSION TITLE
fzf and cursor colors in file picker

### DISCUSSION THREAD

**Discussion by shartf** [LazyVim/LazyVim#3645] [Q&A] [answered]
Hi, 
I've switched to the fzf plugin and for me the colors in the files picker (<leader>space) are not visible:
<img width="550" alt="Screenshot 2024-06-14 at 11 13 23" src="https://github.com/LazyVim/LazyVim/assets/10035382/1c7736ca-0b7b-438a-b6a2-71945ce42910">

The theme I am using is deepwhite: 
```{
    "Verf/deepwhite.nvim",
    lazy = false,
    priority = 1000,
    config = function()
      vim.cmd([[colorscheme deepwhite]])
    end,

    opts = function()
      require("deepwhite").setup({
        low_blue_light = true, 
      })
    end,
  },
```

Now, before I bugreport the author of deepwhite, I would like to understand what is happening there. I assume that deepwhite's cursor bg/fg are somehow to blame there. But if I do following in config:
```
  {
    "ibhagwan/fzf-lua",
    config = function()
      require("fzf-lua").setup({})
    end,
  },
```
I get a very different view:
<img width="483" alt="Screenshot 2024-06-14 at 11 18 57" src="https://github.com/LazyVim/LazyVim/assets/10035382/a06d6997-68c4-4339-89ea-e8701b668f88">

I assume that it resets the LazyVim config for fzf-lua and then the cursor line seems to be visible, even if ugly because not properly supported by the theme? 

In the end I think it is the only visual glitch I've found with fzf-lua and if I could just somehow override bg and fg colors for that case I would already be quite happy...

**Comment by folke** [comment, marked as answer]
cursor colors are managed by the terminal
